pennylane/qchem/integrals.py

```diff
@@ -708,7 +711,7 @@ def _boys(n, t):
     Returns:
         (array[float]): value of the Boys function
     """
-    return anp.where(
+    return np.where(
```

Comment:
If you can end up with a Jax object here, it would be better to replace it with math.

Reply:
Done.